macOS/SurveyAdmin/ViewController.swift

```diff
@@ -23,7 +23,7 @@ struct SurveyResult {
     let noCount: Int
```

Comment:
Sorry, messy commit.  I didn't realized I had that one in there.  It's for the MacOS version of the admin app, you can disregard this, or I can revert/commit the unchanged version to not make it part of this PR.